Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM ubuntu:focal-20221130
```

Comment:
Could you be so kind and use `ubuntu:focal-20230126` as the underlying base image?